conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps_find_module_and_config.py

```diff
@@ -115,10 +115,11 @@ def test_reuse_with_modules_and_config(client):
 
 find_modes = [
     ("both", "both", ""),
+    ("both", "both", "MODULE"),
```

Comment:
This is the new test case being added, where were would expect the generated `FindPKGB.cmake` to find PKGA using the module name specified by PKGA (which was not the case)